src/pytest/test.py

```diff
@@ -2698,3 +2844,34 @@ def grouper(iterable, n, fillvalue=None):
 
 def to_dict(r):
     return {r[i]: r[i + 1] for i in range(0, len(r), 2)}
+
+def testOptimize(env):
```

Comment:
Why not just remove 'optimize'... I don't think we even publish it anywhere

Reply:
For now, it gives some coverage.
Why now remove it when we remove the code as well.